src/bindings/python/src/pyopenvino/graph/dict_attribute_visitor.cpp

```diff
@@ -262,7 +262,7 @@ void util::DictAttributeDeserializer::on_adapter(const std::string& name,
             auto body = std::make_shared<ov::Model>(body_outputs, body_parameters);
             adapter.set(body);
         } else {
-            NGRAPH_CHECK(false, "No AttributeVisitor support for accessing attribute named: ", name);
+            OPENVINO_ASSERT(false, "No AttributeVisitor support for accessing attribute named: ", name);
```

Comment:
```suggestion
            OPENVINO_THROW("No AttributeVisitor support for accessing attribute named: ", name);
```
and in other places as well

Reply:
done